### PR TITLE
feat(generic-metrics): Add gauges to docker compose file

### PR DESCRIPTION
**Pull request by ayirr7**
This was missing by default, adding this consumer to the docker-compose file as well

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -283,6 +287,9 @@ services:
   snuba-generic-metrics-counters-consumer:
     <<: *snuba_defaults
     command: consumer --storage generic_metrics_counters_raw --consumer-group snuba-gen-metrics-counters-consumers --auto-offset-reset=latest --max-batch-time-ms 750 --no-strict-offset-reset
+  snuba-generic-metrics-gauges-consumer:
```
Comment (hubertdeng123) [inline docker-compose.yml:290]: is this a new consumer that was missed? 😅

**Review by hubertdeng123 — APPROVED** [review]

**Review by aldy505 — COMMENTED** [review]
I know that this is missing and as far as I know, this is only used by the DDM backend? If that's the case, can we hold this until DDM is at least in beta or GA?

Adding a new container to the compose file is... an undesirable thing to do for self-hosted Sentry.

**Comment by hubertdeng123** [comment]
If this is only used for DDM, I agree with @aldy505 that we should hold off on this for now

**Review by hubertdeng123 — COMMENTED** [review]
If this is only used for DDM, I agree with @aldy505 that we should hold off on this for now

**Comment by ale-cota** [comment]
Since a lot of people are bound to run into this issue while trying to run metrics on self-hosted, would it be possible/acceptable to add it as a comment at least? So as to inform users that if they want to use DDM, they need to add the additional consumer to their docker-compose file.

**Comment by aldy505** [comment]
> Since a lot of people are bound to run into this issue while trying to run metrics on self-hosted, would it be possible/acceptable to add it as a comment at least? So as to inform users that if they want to use DDM, they need to add the additional consumer to their docker-compose file.

@ale-cota The self-hosted community never do that. For new features we always put an issue and point people to run/enable some things on their config in order to enable a new feature. For this specific issue, it's here: https://github.com/getsentry/self-hosted/issues/2698

For other features, see https://github.com/getsentry/self-hosted/issues/2646, https://github.com/getsentry/self-hosted/issues/1993#issuecomment-1675633779, and.. I forgot where is the issue for Performance Queries

**Comment by chadwhitacre** [comment]
> to inform users that if they want to use DDM, they need to add the additional consumer to their docker-compose file

I like using a GitHub issue for this. Let's start formalizing that as our process. I've added [a `Type: Pre-release Feature` label](https://github.com/getsentry/self-hosted/labels/Type%3A%20Pre-release%20Feature) to this repo and applied it to https://github.com/getsentry/self-hosted/issues/2698. I think it'd be great to have a paragraph on our [self-hosted docs](https://develop.sentry.dev/self-hosted/) about enabling pre-release features and links out to the issue label listing for current pre-release features. We have some reference under [feature flags](https://develop.sentry.dev/feature-flags/) but nothing specific under [self-hosted](https://develop.sentry.dev/self-hosted/) afaict.

@ale Can you help us share https://github.com/getsentry/self-hosted/issues/2698 with customers who are interested trying out Metrics in self-hosted?

@ayirr7 I'm gonna close this PR. 🙏

**Pull request closed without merging**